packages/preview/modern-unimi-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "modern-unimi-thesis"
```

Comment:
Given that there is another package called `modern-unimib-thesis` for another university in Milano, I would suggest to change "modern" for something else (you can have a look at other package names for inspiration if necessary), even if the current name technically complies with our naming guidelines. I feel like it would be too easy to mistype one for the other and use the wrong template.